docs/docs/cmd/tenant/auditlog/auditlog-report.md

```diff
@@ -0,0 +1,61 @@
+# tenant auditlog report
+
+Gets audit logs from the Office 365 Management API
+
+## Usage
+
+```sh
+m365 tenant auditlog report [options]
+```
+
+## Options
+
+`-c, --contentType <contentType>`
+: Audit content type of logs to be retrieved, should be one of the following: AzureActiveDirectory, Exchange, SharePoint, General, DLP.
+
+`-s, --startTime [startTime]`
+: Start time of logs to be retrieved. Start time and end time must be less than or equal to 24 hours apart. Start time is mandatory if End time is specified.
+
+`-e, --endTime [endTime]`
+: End time of logs to be retrieved. Start time and end time must be less than or equal to 24 hours apart. If End time is not specified, command will assume the End time to be 24 hours from the specified Start time.
+
+--8<-- "docs/cmd/_global.md"
+
+## Remarks
+
+By default, if startTime and endTime are not mentioned, then the content available in the last **24 hours** is returned. `StartTime` and `EndTime` must be less than or equal to **24 hours** apart, with the `StartTime` prior to `EndTime` and `StartTime` no more than 7 days in the past.
```

Comment:
Since `StartTime` and `EndTime` refer to options, let's use the same casing (`startTime` and `endTime`)